src/book-demo.njk

```diff
@@ -1,18 +1,20 @@
 ---
-layout: layouts/nohero.njk
+layout: layouts/page.njk
 title: Book a demo
 sitemapPriority: 0.8
 image: /images/og-book-demo.jpeg
 meta:
     description: Book a demo to understand how FlowForge helps you achieve your goals.
```

Comment:
@ZJvandeWeg should we change this text? "achieve your goals" seems too generic... But I'm not sure what we could say instead

Reply:
```suggestion
    description: Book a demo to understand how FlowForge helps you professionalize your Node-RED deployment.
```

Reply:
Not certain of my proposal, but it's something?